apps/api/src/donations/donations.service.ts

```diff
@@ -325,14 +325,13 @@ export class DonationsService {
           include: { campaign: { select: { title: true } } },
         },
       },
-      where: { person: { keycloakId } },
+      where: { person: { keycloakId }, status: DonationStatus.succeeded },
```

Comment:
If we filter on backend, this will hide the donations with statuses canceled from the User Profile donations table. Could you make the filter on the frontend only for the summary?